js/pihole/footer.js

```diff
@@ -33,18 +33,18 @@ function versionCompare(left, right) {
 
 // Update check
 $.getJSON("https://api.github.com/repos/pi-hole/pi-hole/releases/latest", function(json) {
-    // Skip if on dev
-    if(piholeVersion !== "vDev" && versionCompare(piholeVersion, json.tag_name.slice(1)) < 0) {
-        // Alert user
-        $("#piholeVersion").html($("#piholeVersion").text() + '<a class="alert-link" href="https://github.com/pi-hole/pi-hole/releases">(Update available!)</a>');
+    if(webVersion !== "vDev" && piholeVersion !== "Stub" && versionCompare(webVersion, json.tag_name.slice(1)) < 0) {
```

Comment:
Change `webVersion` to `piholeVersion` in the `vDev` check and in `versionCompare`, probably copy-paste error.